PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -33,333 +33,11 @@
         <li>Subscription ID need to be specified for ADLS Gen 1 storage type</li>
       </ul>
       <h4>Fixed</h4>
+      <ul>
         <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2840" rel="nofollow">#2840</a> Submit successfully with invalid path for WebHDFS storage type issue.</li>
         <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2747" rel="nofollow">#2747</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2801" rel="nofollow">#2801</a> Error loadig HDInsight node issue.</li>
         <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2714" rel="nofollow">#2714</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2688" rel="nofollow">#2688</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2669" rel="nofollow">#2669</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2728" rel="nofollow">#2728</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2807" rel="nofollow">#2807</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2808" rel="nofollow">#2808</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2811" rel="nofollow">#2811</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2831" rel="nofollow">#2831</a>Spark Run Configuration validation issues.</li>
         <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2810" rel="nofollow">#2810</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2760" rel="nofollow">#2760</a> Spark Run Configuration issues when created from context menu.</li>
```

Comment:
Can we have a line at bottom to link to the `CHANGELOG.md`?

Reply:
yes, added a link at the bottom of this page